day2/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import logo from './logo.png';
+import './App.css';
+
+const quotes = ["Stop crying. Fucking code.", "Knock Knock. Who's There? Fuck you. Work.", "Show some fucking heart",
+        "React isn't that fucking hard, just do it.", "Eat some healthy fucking food.", "Try being a little fucking social.",
+        "Fucking smile.", "Make people fucking like you."]
+
+var x = 0;
+var y = quotes.length;
+var arrayIndex = x;
```

Comment:
few things here. `x` and `y` give no bearing as to what they are. also, this should most likely just be accomplished with state. `arrayIndex` should also be just managed in state and not as a global variable